### ISSUE TITLE
Replicating COCO model

### ISSUE THREAD

**Issue by matttrent** [NVlabs/SPADE#51]
I'm trying to train SPADE on the COCO dataset, and reproduce the FID numbers.  I've already managed to replicate the FID score for the model trained with ADE20K, and just want to confirm a few of the settings.  It's a bit unclear from Appendix A how the COCO settings differ from the ADE20K ones.  So:

1. I see the number of epochs is 100.  For COCO, is the learning rate schedule constant for all 100, or is it constant for the first 50 and for the second 50 it linearly decays to zero?
1. `--batchSize` is still 32 (assuming 8 GPUs)?
1. `--coco_no_portraits` should not be set (False)?
1. `--use_vae` should not be set (False)?

Any other setting differences I should be aware of?

Thanks again for all the helpful replies.

**Comment by Cold-Winter** [comment]
Can you please give me some kinds of source code for calculating the FID numbers. I have found one here https://github.com/NVlabs/stylegan/blob/master/metrics/frechet_inception_distance.py. Do you use the same one.

**Comment by matttrent** [comment]
I used the code from this repo:  https://github.com/mseitzer/pytorch-fid

The numbers it produced for both ADE20 and COCO Stuff match what's published in the paper.

**Comment by Cold-Winter** [comment]
Thanks for your response. I test this code on cityscapes and I get 63 FID which is much better than the FID score reported in their paper which is 71. Do you reproduce these results? When we test with FID score, should we rescale the image as 299*299 which is the input of the inception net?

**Comment by Hshihua** [comment]
@matttrent When you calculate the FID score, you take the training samples or the validation samples as the real images? I am confused as I got 32.12 or 39.65 when I took the resized training samples or the resized validation samples as the input, respectively.
